tests/conftest.py

```diff
@@ -210,6 +210,8 @@ def users(UserFixture, app, database, users_data):
         u.create(app, database)
         users[obj["username"]] = u
     UserAggregate.index.refresh()
```

Comment:
Very minor: Since we use `current_...` we can replace it here as well.
```suggestion
    current_users_service.record_cls.index.refresh()
```